client/beefy/src/import.rs

```diff
@@ -0,0 +1,195 @@
+use std::{collections::HashMap, marker::PhantomData, sync::Arc};
+
+use sc_consensus::{
+	BlockCheckParams, BlockImport, BlockImportParams, ImportResult, JustificationImport,
+};
+
+use sc_client_api::backend::Backend;
+use sp_api::{ProvideRuntimeApi, TransactionFor};
+use sp_blockchain::well_known_cache_keys;
+use sp_consensus::Error as ConsensusError;
+use sp_runtime::{
+	generic::{BlockId, OpaqueDigestItemId},
+	traits::{Block as BlockT, Header as HeaderT, NumberFor},
+	Justification,
+};
+
+use beefy_primitives::{
+	crypto::{AuthorityId, Signature},
+	BeefyApi, ConsensusLog, ValidatorSet, BEEFY_ENGINE_ID,
+};
+
+use crate::{
+	justification::decode_and_verify_justification, notification::BeefyJustificationSender,
+	Client as BeefyClient,
+};
+
+/// Checks the given header for a consensus digest signalling a beefy authority set change
+/// and extracts it.
+pub fn find_beefy_authority_set_change<B: BlockT>(
```

Comment:
no reason for this helper fn to be public

```suggestion
fn find_beefy_authority_set_change<B: BlockT>(
```